### DISCUSSION TITLE
Backups

### DISCUSSION THREAD

**Discussion by justinwaltrip** [afadil/wealthfolio#126] [General]
Before I spend too much time adding/updating my accounts, I want to have a backup solution in place so that I can restore this effort in the event of data loss or moving to another system. I see that an export feature is in the roadmap, but I am wondering if there is a way to setup backup/restore before then.

**Comment by afadil** [comment]
Now you can export all your data including the SQLite full backup. Available in v1.018
![Export Feature](https://github.com/user-attachments/assets/925c90c6-3595-48ba-a057-29bc34dc3587)

**Reply by LaPeste** [reply]
How can a backup be reimported? I can't seem to find an import button.

EDIT:
I saw through the logs. It's the first line.
For whoever comes here, if on Windows, just replace `$HOME\AppData\Roaming\com.teymz.wealthfolio\app.db` with your backup and rename it to `app.db`

**Reply by afadil** [reply]
There is now a Restore Button in both the Desktop and Mobile apps. In the Docker web version, restoration must be done manually.

**Reply by Billoux78** [reply]
> There is now a Restore Button in both the Desktop and Mobile apps. In the Docker web version, restoration must be done manually.

Hi Afadil, thank you for your great job !
I am testing Wealthfolio since a week in a Windows version, and I would like to import my backup in my new Docker version.
You have written that "In the Docker web version, restoration must be done manually.", but nowhere I found how to make it.
Can you give more explanation, please ?
Thank's !
